src/app/common/components/MustGatherModal.tsx

```diff
@@ -39,32 +40,31 @@ export const MustGatherModal: React.FunctionComponent = () => {
   );
 
   const handleMustGatherRequest = ({ customName, type }: MustGatherObjType) => {
+    const namespacedName = withNs(customName, type);
     registerMustGather.mutate({
-      'custom-name': customName,
+      'custom-name': namespacedName,
       command:
         type === 'plan'
-          ? `PLAN=${customName} /usr/bin/targeted`
-          : `VM=${customName} /usr/bin/targeted`,
+          ? `PLAN=${namespacedName} /usr/bin/targeted`
```

Comment:
Should this be the `customName` instead of `namespacedName`? Wouldn't the service need the bare plan/VM name?

Reply:
The must gather service will use whatever you supply, it's not required to use any specific format. If we don't include the namespacing info we'll have to re-stitch the values back together elsewhere in the UI where we do lookups. If there's no issue with including the namespace, I think it may be better to keep it in lock step with what the backend receives.